tests/bootstrap.php

```diff
@@ -1,7 +1,60 @@
 <?php
 
-$loader = @include __DIR__ . '/../vendor/autoload.php';
-if (!$loader) {
-    $loader = require __DIR__ . '/../../../../vendor/autoload.php';
+namespace React\Tests\Dns;
+
+use React\Promise\Deferred;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+class TestCase extends \PHPUnit_Framework_TestCase
```

Comment:
Any particular reason to place `TestCase` in the bootstrap rather then it's own file?